kms/checkStateImportedKey.js

```diff
@@ -0,0 +1,75 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+async function main(
+  projectId = 'my-project',
+  locationId = 'us-east1',
+  keyRingId = 'my-key-ring',
+  cryptoKeyId = 'my-imported-key',
+  cryptoKeyVersionId = '1'
+) {
+  // [START kms_check_state_imported_key]
+  //
+  // TODO(developer): Uncomment these variables before running the sample.
+  //
+  // const projectId = 'my-project';
+  // const locationId = 'us-east1';
+  // const keyRingId = 'my-key-ring';
+  // const cryptoKeyId = 'my-imported-key';
+  // const cryptoKeyVersionId = '1';
```

Comment:
Can we add a docs link on where to get more information on the params?

Reply:
These samples are meant to be embedded within https://cloud.google.com/kms/docs/importing-a-key - there is already fairly detailed textual description for all of these parameters that is not language-specific.  (Or are you just asking for more context /outside/ the samples?